layouts/partials/header.html

```diff
@@ -24,6 +24,7 @@
 
             <span class="nav-icons-divider"></span>
             <div class="nav-link dark-theme-toggle">
+                <span id="dark-theme-toggle-screen-reader-target" class="sr-only">theme</span>
```

Comment:
Please use the classname `screen-reader`

Reply:
`sr-only` is an industry standard for this kind of thing. Is there a specific reason to use that? Was that funcionality already implemented and I jsut missed it?